src/test_l1_dsVideoDevice.c

```diff
@@ -890,26 +886,29 @@ void test_l1_dsVideoDevice_positive_dsGetVideoCodecInfo(void)
     UT_ASSERT_EQUAL(result, dsERR_NONE);
 
     // Step 02: Obtain video device handle
-    result = dsGetVideoDevice(index, &handle);
-    UT_ASSERT_EQUAL(result, dsERR_NONE);
-    UT_ASSERT_EQUAL((handle >= 0), true);
-
-    // Step 03: Get video codec information with dsVIDEO_CODEC_MPEGHPART2
-    result = dsGetVideoCodecInfo(handle, dsVIDEO_CODEC_MPEGHPART2, &codecInfo);
-    UT_ASSERT_EQUAL(result, dsERR_NONE);
+    for(int i = 0; i < gDSvideoDevice_NumVideoDevices; i++){
+        result = dsGetVideoDevice(index, &handle);
+        UT_ASSERT_EQUAL(result, dsERR_NONE);
+        UT_ASSERT_EQUAL((handle >= 0), true);
 
-    // Step 04: Get video codec information with dsVIDEO_CODEC_MPEG4PART10
-    result = dsGetVideoCodecInfo(handle, dsVIDEO_CODEC_MPEG4PART10, &codecInfo);
-    UT_ASSERT_EQUAL(result, dsERR_NONE);
+        // Step 03: Get video codec information with dsVIDEO_CODEC_MPEGHPART2
```

Comment:
use for loop to check the values and compare with profile file (check L2 file )
kvp now support the float values so that change also

Reply:
done